pack.properties

```diff
@@ -0,0 +1,4 @@
+displayName='Memberberries
```

Comment:
Remove the ' as it might cause issues

Reply:
Done deal, hopefully all is well with it :)